message-index/messages/GHC-51179/example1/after/ErrLambdaCase.hs

```diff
@@ -0,0 +1,4 @@
+{-# LANGUAGE LambdaCase #-}
+module ErrLambdaCase where
+
+f = \case "1" -> 1
```

Comment:
This might give the impression that lambda-case is only for a single match, and not when we want to eta-reduce or avoid naming parameters.

Reply:
Good point! What about this?
```suggestion
f = \case
      "1" -> 1
      "2" -> 2
      _ -> -1
```

Reply:
(and a suitable update to the error message)

Reply:
OK, changed example to:
```
f = \case {"1" -> 1; "2" -> 2}
```

Is this enough or should I think of more elaborate example?

Reply:
The example illustrates the point better now, but the braces-and-semicolon syntax is not particularly idiomatic Haskell. I worry that people will think that this syntax is common.